extensions/big-data-cluster/package.nls.json

```diff
@@ -0,0 +1,9 @@
+{
+	"displayName": "SQL Server Big Data Clusters",
+	"description": "Support for managing SQL Server Big Data Clusters",
+	"explorer.title": "SQL Server Big Data Clusters",
```

Comment:
As mentioned in previous review, if all these are identical `"SQL Server Big Data Clusters"` let's just have 1 variable. That saves the localization folks work.

Reply:
fixed.